4_Zadania_dodatkowe/6.InfiniteSlider/js/app.js

```diff
@@ -0,0 +1,71 @@
+$(document).ready(function () {
+   // console.log('test');
+
+    var elementPrevButton = $('#prevPicture');
+    var elementNextButton = $('#nextPicture');
+    // console.log(elementPrevButton, elementNextButton);
+
+    var elementListLi = $('li');
+    // console.log(elementListLi);
+
+    var elementUl = $('ul');
+    // console.log(elementUl);
+
+    var visibleImageIndex = 1;
+    var imageWidth = 404;
+    // console.log(visibleImageIndex, imageWidth);
+
+    var firstElement = elementListLi.first().clone();
+    var lastElement = elementListLi.last().clone();
+    // console.log(firstElement);
+    // console.log(lastElement);
+    elementUl.append(firstElement);
+    elementUl.prepend(lastElement);
```

Comment:
this is the right time to handle position of `ul` element so first image is visible and not the one you just cloned and prepended

please notice that when you load the website you may see rhinos image for a fraction of second (this is because `js` is not executed immediately), this is an unpleasant visual effect

Reply:
updated in the newest commit